addon/mixins/transition-mixin.js

```diff
@@ -26,203 +75,164 @@ export default Mixin.create({
     return this.get('transitionClasses').join(' ');
   }),
 
-  addClass(className, $element) {
+  addClass(className, element) {
     if (!this.get('isDestroying')) {
-      run(() => {
-        this.get('transitionClasses').addObject(className);
-      });
+      this.get('transitionClasses').addObject(className);
```

Comment:
And if so, this line would need it too.

https://github.com/peec/ember-css-transitions/blob/278c6699f088b2289c4cf781328cf54d69575a63/addon/mixins/transition-mixin.js#L232